### PR TITLE
Fixes video and admin pack images

### LINKED ISSUE #10036: Welcome page videos not showing when two welcome screens are open.
- Azure Data Studio Version:

Steps to Reproduce:

1. Open ADS
2. Split welcome page screen
3. Notice the videos are not showing on the 2nd welcome page

### LINKED ISSUE #10037: SQL Admin Pack background images needs to be replaced
- Azure Data Studio Version:

The SQL Admin Pack background image needs to replaced with an illustration rather than a photo.

### PR DESCRIPTION
**Pull request by v-bbrady**
This PR fixes #10037 
This PR fixes #10036

### CODE REVIEW

File: src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts [hunk 1]
```diff
@@ -322,11 +322,14 @@ class WelcomePage extends Disposable {
 	}
 
 	private addVideoImageSource() {
-		const videoIntroduction = document.querySelector('#video_introduction') as HTMLImageElement;
-		const videoOverview = document.querySelector('#video_overview') as HTMLImageElement;
-
-		videoIntroduction.src = require.toUrl('./../../media/video_introduction.png');
-		videoOverview.src = require.toUrl('./../../media/video_overview.png');
+		const videoIntroduction = document.querySelectorAll('.video_introduction') as NodeListOf<HTMLImageElement>;
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts:325]: Is there a reason you wouldn't just embed this directly into the az_data_welcome_page directly?

Comment (v-bbrady) [reply src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts]: No reason other than i was originally having issues with finding the relative path from the az_data_welcome_page to the media folder. It's fixed now.

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 2]
```diff
@@ -165,15 +165,15 @@ export default () => `
 							<div class="flex flex_container_video">
 								<div class="videos_container_video">
 									<a href="https://www.youtube.com/watch?v=Orv7fptVoUA" class="video overview">
-									<img class="video_overview" id="video_overview" />
+									<img src="./../../../../sql/workbench/contrib/welcome/media/video_overview.png" class="video_overview" id="video_overview" />
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:168]: You should just embed the require.toUrl call into here like you do with the localized strings

Comment (v-bbrady) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: done

**Pull request merged**